### PR TITLE
Add timeout to all HTTP requests

### PR DESCRIPTION
**Pull request by klochowicz**
Reqwest doesn't add request timeout by default.

### CODE REVIEW

File: maker/src/trading/orderbook_client.rs [hunk 1]
```diff
@@ -12,7 +12,10 @@ pub struct OrderbookClient {
 impl OrderbookClient {
     pub fn new() -> Self {
         Self {
-            client: reqwest::Client::new(),
+            client: reqwest::Client::builder()
+                .timeout(std::time::Duration::from_secs(10))
```
Comment (holzeis) [inline maker/src/trading/orderbook_client.rs:16]: 🤔 is 10 seconds conservative enough? e.g. the generation of the fake scid takes sometimes more than 10 seconds. how about 30 seconds to be on the safe side?

**Review by holzeis — APPROVED** [review]
LGTM 👍

File: maker/src/trading/orderbook_client.rs (hunk 1 above)
Comment (klochowicz) [reply maker/src/trading/orderbook_client.rs:16]: > 🤔 is 10 seconds conservative enough? e.g. I haven't stopped the time but the generation of the fake scid takes sometimes very long. how about 30 seconds to be on the safe side?

Sure, 10s was arbitrary! Shall we put 30s everywhere to be safe? or were you just talking about this place?

Comment (holzeis) [reply maker/src/trading/orderbook_client.rs:16]: Yes, let's go for 30 seconds to be safe. 😊

**Pull request merged**